src/entity/Company.java

```diff
@@ -1,35 +1,36 @@
 package entity;
 
-import java.util.ArrayList;
+import parser.CarsFileReader;
+import parser.CarsParser;
+
 import java.util.List;
-import java.util.logging.Level;
-import java.util.logging.Logger;
 import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.concurrent.locks.Lock;
 import java.util.concurrent.locks.ReentrantLock;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
 public class Company {
     private static Company company;
-    private static AtomicBoolean instanceCreated = new AtomicBoolean(false);
+    private static AtomicBoolean instanceCreated = new AtomicBoolean();
     private static Lock lock = new ReentrantLock();
-    private static final int COUNT_OF_CARS = 10;
-    private List<Car> cars = new ArrayList<>();
+    private static final String FILE_PATH = "cars.csv";
+    private static final List<Car> CARS = new CarsParser().getCars(new CarsFileReader().parseCarList(FILE_PATH));
```

Comment:
Move this logic to private method and call its inside getInstance(). You shouldn't have any logic here. Only definitions.